# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/test/webgl-conf/generated-mochitest.ini
# Commit: 7e2a2b1b416f
# Full Hash: 7e2a2b1b416f1a64c6548aab29e92acc9554ae7d
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Fix all webgl regression tests according to CI. r=handyman
#   
#   (This is a combination of 31 commits)
#   
#   * Fix Linux compilation.
# ==============================================================================

diff -r ccfa767dba64 -r 7e2a2b1b416f dom/canvas/test/webgl-conf/generated-mochitest.ini
--- a/dom/canvas/test/webgl-conf/generated-mochitest.ini	Wed Jan 08 22:19:16 2020 +0000
+++ b/dom/canvas/test/webgl-conf/generated-mochitest.ini	Wed Jan 08 22:19:23 2020 +0000
@@ -5074,8 +5074,8 @@
 subsuite = webgl2-core
 [generated/test_2_conformance2__extensions__ovr_multiview2.html]
 subsuite = webgl2-core
+fail-if = (os == 'win')
 skip-if = (os == 'android')
-fail-if = (os == 'win')
 [generated/test_2_conformance2__extensions__ovr_multiview2_depth.html]
 subsuite = webgl2-core
 skip-if = (os == 'android')
@@ -5091,8 +5091,8 @@
 subsuite = webgl2-core
 [generated/test_2_conformance2__extensions__ovr_multiview2_single_view_operations.html]
 subsuite = webgl2-core
+fail-if = (os == 'win')
 skip-if = (os == 'android')
-fail-if = (os == 'win')
 [generated/test_2_conformance2__extensions__ovr_multiview2_timer_query.html]
 subsuite = webgl2-core
 [generated/test_2_conformance2__extensions__ovr_multiview2_transform_feedback.html]
@@ -5247,7 +5247,6 @@
 skip-if = (os == 'win')
 [generated/test_2_conformance2__query__query.html]
 subsuite = webgl2-core
-skip-if = (os == 'win')
 [generated/test_2_conformance2__reading__format-r11f-g11f-b10f.html]
 subsuite = webgl2-core
 fail-if = (os == "win" && processor == "aarch64") #aarch64 due to bug 1536182
@@ -5343,8 +5342,8 @@
 fail-if = (os == 'win')
 [generated/test_2_conformance2__rendering__framebuffer-texture-level1.html]
 subsuite = webgl2-core
+fail-if = (os == 'mac')
 skip-if = (os == 'linux')
-fail-if = (os == 'mac')
 [generated/test_2_conformance2__rendering__framebuffer-unsupported.html]
 subsuite = webgl2-core
 [generated/test_2_conformance2__rendering__fs-color-type-mismatch-color-buffer-type.html]
@@ -7556,7 +7555,6 @@
 subsuite = webgl2-core
 [generated/test_2_conformance__extensions__webgl-compressed-texture-s3tc.html]
 subsuite = webgl2-core
-fail-if = (os != 'android')
 [generated/test_2_conformance__extensions__webgl-compressed-texture-size-limit.html]
 subsuite = webgl2-core
 skip-if = (os == 'win' && os_version == '6.1')
@@ -8753,8 +8751,8 @@
 subsuite = webgl2-core
 [generated/test_2_conformance__rendering__rendering-stencil-large-viewport.html]
 subsuite = webgl2-core
+fail-if = (os == 'mac')
 skip-if = (os == 'android') || (os == 'win')
-fail-if = (os == 'mac')
 [generated/test_2_conformance__rendering__scissor-rect-repeated-rendering.html]
 subsuite = webgl2-core
 [generated/test_2_conformance__rendering__simple.html]
@@ -11451,8 +11449,8 @@
 subsuite = webgl1-ext
 [generated/test_conformance__glsl__misc__shader-with-non-reserved-words.html]
 subsuite = webgl1-ext
+fail-if = (os == 'android')
 skip-if = 1
-fail-if = (os == 'android')
 [generated/test_conformance__glsl__misc__shader-with-precision.frag.html]
 subsuite = webgl1-ext
 [generated/test_conformance__glsl__misc__shader-with-preprocessor-whitespace.html]
@@ -11627,8 +11625,8 @@
 skip-if = (os == 'android')
 [generated/test_conformance__misc__type-conversion-test.html]
 subsuite = webgl1-core
+fail-if = (os == 'linux')
 skip-if = (os == 'android') || (os == 'linux')
-fail-if = (os == 'linux')
 [generated/test_conformance__misc__uninitialized-test.html]
 subsuite = webgl1-core
 skip-if = (os == 'android')
@@ -12182,8 +12180,8 @@
 subsuite = webgl1-core
 [generated/test_conformance__rendering__rendering-stencil-large-viewport.html]
 subsuite = webgl1-core
+fail-if = (os == 'mac') || (verify && debug && (os == 'linux'))
 skip-if = (os == 'android') || (os == 'win')
-fail-if = (os == 'mac') || (verify && debug && (os == 'linux'))
 [generated/test_conformance__rendering__scissor-rect-repeated-rendering.html]
 subsuite = webgl1-core
 [generated/test_conformance__rendering__simple.html]
@@ -12454,8 +12452,8 @@
 subsuite = webgl1-core
 [generated/test_conformance__textures__misc__texture-corner-case-videos.html]
 subsuite = webgl1-core
+fail-if = (os == 'linux' && os_version == '18.04')
 skip-if = (os == 'android')
-fail-if = (os == 'linux' && os_version == '18.04')
 [generated/test_conformance__textures__misc__texture-cube-as-fbo-attachment.html]
 subsuite = webgl1-core
 [generated/test_conformance__textures__misc__texture-draw-with-2d-and-cube.html]